wishful_upis/radio.py

```diff
@@ -24,6 +24,7 @@
 ]
 
 
+
 # Generic API to control the lower layers, i.e. key-value configuration.
 def set_parameters(param_key_values_dict):
```

Comment:
This seens like a workaround on this issue (https://github.com/wishful-project/framework/issues/1). 
This upi does the same as [set_parameter_lower_layer(**kwargs)](https://github.com/wishful-project/wishful_upis/blob/master/wishful_upis/radio.py#L25)?